core/page-assets/loader.php

```diff
@@ -35,11 +35,29 @@ private function init_assets() {
 					'version' => ELEMENTOR_VERSION,
 					'dependencies' => [],
 				],
+				'e-swiper' => [
+					'src' => $this->get_css_assets_url( 'e-swiper', 'assets/css/conditionals/' ),
+					'version' => ELEMENTOR_VERSION,
+					'dependencies' => [],
```

Comment:
Not sure, but shouldn't we add `swiper` as a dependency for `e-swiper`?

Reply:
I love it.